config/packages/security.yaml

```diff
@@ -29,3 +29,7 @@ security:
         - { path: ^/saml, role: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/bi/, role: ROLE_BI }
         - { path: ^/, role: ROLE_USER }
+
+    access_decision_manager:
+        strategy: unanimous
+        allow_if_all_abstain: false
```

Comment:
This configuration does not seem related to anything else in this pull request. Is it added by mistake?

Reply:
I introduce a new voter in the core bundle that is used alongside another voter. So to make sure that every voter counts, this setting is made.

Reply:
https://symfony.com/doc/current/security/voters.html#changing-the-access-decision-strategy

Reply:
Ah, there is a related PR (https://github.com/aakb/kontrolgruppen-core-bundle/pull/232)!